integration-tests/sample_test.go

```diff
@@ -27,6 +28,11 @@ func TestAuth(t *testing.T) {
 		t.Fatalf("Error making initial request to the home page: %v", err)
 	}
 
+	_, err = url.ParseRequestURI(string(authorizeRequestURL))
+	if err != nil {
+		t.Fatalf("Home page didn't respond with authorize url, actual respond: %v", string(authorizeRequestURL))
```

Comment:
Let's have the err in the message so we know if there is some other problem.

Reply:
done